helm/tenant/values.yaml

```diff
@@ -1,189 +1,339 @@
-## Secret with default environment variable configurations to be used by MinIO Tenant.
-## Not recommended for production deployments! Create the secret manually instead.
+###
+# Root key for dynamically creating a secret for use with configuring root MinIO User
+# Specify the ``name`` and then a list of environment variables. 
+#
+# .. important::
+#
+#    Do not use this in production environments. 
+#    This field is intended for use with rapid development or testing only.
+# 
+# For example:
+#
+# .. code-block:: yaml
+#
+#    name: myminio-env-configuration
+#    accessKey: minio
+#    secretKey: minio123
+#
 secrets:
   name: myminio-env-configuration
-  # MinIO root user and password
-  accessKey: minio
+  accessKey: minio 
   secretKey: minio123
-  ## Set the value for existingSecret to use a pre created secret and dont create default one
-  # existingSecret: random-env-configuration
-## MinIO Tenant Definition
+###
+# The name of an existing Kubernetes secret to import to the MinIO Tenant
+# The secret must contain a key ``config.env``.
+# The values should be a series of export statements to set environment variables for the Tenant.
+# For example:
+#
+# .. code-block:: shell
+#
+#    stringData:
+#       config.env: | -
+#         export MINIO_ROOT_USER=ROOTUSERNAME
+#         export MINIO_ROOT_PASSWORD=ROOTUSERPASSWORD
+#
+existingSecret:
+  name: myminio-env-configuration
```

Comment:
Please revise this change. I couldn't find any usage of `.Values.existingSecret` in tenant Helm templates. There's usage of `.Values.secrets.existingSecret` instead.